content/pages/FAQ.rst

```diff
@@ -85,7 +84,9 @@ Where do I park?
 Free parking is widely available on campus on Saturdays, with the
 exception of reserved and metered stalls. There is a large parking lot
 to the north of Kelly Engineering that should have enough room for
-everyone.
+everyone. A detailed map can be found on the `About page`_.
+
+.. _About page: http://beaverbarcamp.org/pages/about.html
```

Comment:
I wouldn't hard link the website here, I'd just change this to ``/pages/about.html`` so it would work locally or in production.